src/routes/(app)/admin[[hash=admin_hash]]/reporting/+page.server.ts

```diff
@@ -1,15 +1,36 @@
 import { collections } from '$lib/server/database';
 import { countryFromIp } from '$lib/server/geoip';
-import { pojo } from '$lib/server/pojo';
+import { getConfirmationBlocks } from '$lib/server/getConfirmationBlocks';
 import { sum } from '$lib/utils/sum';
+import { toSatoshis } from '$lib/utils/toSatoshis';
 
 export async function load() {
 	const orders = await collections.orders.find().sort({ createdAt: -1 }).toArray();
 
 	return {
 		orders: orders.map((order) => ({
 			_id: order._id,
-			payments: order.payments.map(pojo),
+			payments: order.payments.map((payment) => ({
+				id: payment._id.toString(),
+				method: payment.method,
+				status: payment.status,
+				address: payment.address,
+				expiresAt: payment.expiresAt,
+				paidAt: payment.paidAt,
+				createdAt: payment.createdAt,
+				checkoutId: payment.checkoutId,
+				invoice: payment.invoice,
+				price: payment.price,
+				invoiceId: payment.invoiceId,
+				transactions: payment.transactions,
+				currencySnapshot: payment.currencySnapshot,
+				confirmationBlocksRequired:
+					payment.method === 'bitcoin'
+						? getConfirmationBlocks(toSatoshis(payment.price.amount, payment.price.currency))
+						: 0,
+				...(payment.bankTransferNumber && { bankTransferNumber: payment.bankTransferNumber }),
+				...(payment.detail && { detail: payment.detail })
+			})),
```

Comment:
That's an overreaction, `invoiceNumberVariables` only needs `id: string` on the payments, so you can do this instead:

```
`			payments: order.payments.map(payment => ({...pojo(payment), id: payment._id.toString()})),
```

That, or you can change it in `receipt` and `summary` (to use `_id` instead of `id`)